frontend/api/index.js

```diff
@@ -65,7 +65,10 @@ app.get('/config/project-overrides', (req, res) => {
         { name: 'api', value: process.env.FLAGSMITH_PROXY_API_URL ? '/api/v1/' : process.env.FLAGSMITH_API_URL },
         { name: 'maintenance', value: envToBool('ENABLE_MAINTENANCE_MODE', false) },
         { name: 'flagsmithClientAPI', value: process.env.FLAGSMITH_ON_FLAGSMITH_API_URL },
-        { name: 'disableInflux', value: !envToBool('ENABLE_INFLUXDB_FEATURES', true) },
+        { name: 'disableAnalytics', value: envToBool('DISABLE_ANALYTICS_FEATURES', false) },
+        { name: 'disableInflux', value: !envToBool('ENABLE_INFLUXDB_FEATURES', true) ||
+                envToBool('DISABLE_INFLUXDB_FEATURES', false)
+        },
```

Comment:
Note, this adds DISABLE_INFLUXDB_FEATURES. It was noted that the docs/frontend use ENABLE_INFLUXDB_FEATURES, since we can't just remove that this will allow us to deprecate it/update the docs.